flujo/validation/linters.py

```
@@ -1496,6 +1503,261 @@ def analyze(self, pipeline: Any) -> Iterable[ValidationFinding]:
+                    branches = getattr(st, "branches", {}) or {}
+                    for branch_name, branch_pipeline in branches.items():
+                        if branch_pipeline:
+                            branch_steps = getattr(branch_pipeline, "steps", []) or []
+                            for branch_step in branch_steps:
+                                skill_ref = self._get_custom_skill_ref(branch_step, meta)
+                                if skill_ref:
+                                    has_custom_skills = True
+                                    custom_skill_refs.append(f"{branch_name}:{skill_ref}")
+
+            except Exception:
+                continue
```

Ruff (0.13.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)